app/controllers/group_members_controller.rb

```diff
@@ -0,0 +1,90 @@
+class GroupMembersController < ApplicationController
+  include CollaboratorProcessing
+
+  before_action :find_group_member, only: [:destroy, :make_admin]
+  before_action :check_admin_member_present, only: :destroy
+
+  def create
+    if group_member_params[:user_ids].present?
+      user_ids = group_member_params[:user_ids].split(',')
+    else
+      flash[:warning] = 'At least one user must be added!'
+      redirect_to group_path(group_member_params[:group_id])
+    end
+
+    if user_ids.present?
+      user_ids.each do |user_id|
+        group_member = GroupMember.new(
+          user_id: user_id,
+          group_id: group_member_params[:group_id]
+        )
+        if group_member.save
+          group_resources(group_member).each do |resource|
+            add_users_as_collaborators(resource, group_member.user.id.to_s, group_member.group.id)
+          end
+        else
+          flash[:warning] = group_member.errors.full_messages.join(', ')
+          redirect_to group_path(group_member_params[:group_id])
+          return
```

Comment:
Given that we're iterating over a collection of `user_ids` and saving an object for each, suggest we shovel errors onto the `flash[:warning]` to collect any that fail and allow the iteration to continue.